src/main/java/org/folio/helper/CheckinReceivePiecesHelper.java

```diff
@@ -551,14 +550,17 @@ protected Future<ReceiptStatus> calculatePoLineReceiptStatus(PoLine poLine, List
    * piece records is zero, returns "Awaiting Receipt" status, otherwise -
    * "Partially Received"
    *
-   * @param expectedPiecesQuantity
-   *          expected piece records quantity
+   * @param byPoLine
+   *          all piece records obtained by PO line id
    * @param poLine
    *          PO Line record representation from storage
    * @return completable future holding calculated PO Line's receipt status
    */
-  private Future<ReceiptStatus> calculatePoLineReceiptStatus(int expectedPiecesQuantity, PoLine poLine,
-      List<Piece> pieces, RequestContext requestContext) {
+  private Future<ReceiptStatus> calculatePoLineReceiptStatus(List<Piece> byPoLine, PoLine poLine, List<Piece> pieces) {
+    List<ReceivingStatus> receivingStatuses = List.of(EXPECTED, CLAIM_DELAYED, CLAIM_SENT);
```

Comment:
small improvement here to introduce constant, because these statuses are used in 2 places

Reply:
I was considering this change as well but it appeared that one places uses `org.folio.rest.acq.model.ReceivingStatus` but an nother uses `org.folio.rest.jaxrs.model.ReceivingStatus`

Reply:
Updated to use models from same package and introduced the constant